src/main/components/PianoRoll/PianoRoll.tsx

```diff
@@ -127,8 +128,14 @@ const PianoRollWrapper: FC = observer(() => {
     (e: React.WheelEvent) => {
       if (e.altKey || e.ctrlKey) {
         // zooming
-        const scaleFactor = isTouchPadEvent(e.nativeEvent) ? 0.01 : 0.05
-        s.scaleX = Math.max(SCALE_X_MIN, s.scaleX + e.deltaY * scaleFactor)
+        let scaleFactor = undefined
+        if (navigator.appVersion.indexOf("Mac") != -1) { // MacOS
+          scaleFactor = isTouchPadEvent(e.nativeEvent) ? 0.005 : -0.02
+        } else {
+          scaleFactor = isTouchPadEvent(e.nativeEvent) ? 0.01 : 0.05
+        }
+        const scaleXDelta = clamp(e.deltaY * scaleFactor, -0.2, 0.2) // prevent acceleration to zoom too fast
+        s.scaleAroundPointX(scaleXDelta, e.clientX - Layout.keyWidth)        
```

Comment:
Not sure if `e.clientX` is the best way to get the pointer position on the PianoRoll-canvas, but is seem to work